From ec2963f852ddcfe4836fe9f8f71f574b6265b7af Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 22 Aug 2022 14:58:19 +0200
Subject: [PATCH] libgd: add conflicts to each other

The full variant should conflict with the default variant. This prevents that
libgd and libgd-full could be installed side by side, and also, the full
variant should provide the libgd. Otherwise, if you install libgd-full,
you can not install vnstat.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 libs/libgd/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libs/libgd/Makefile b/libs/libgd/Makefile
index 0e614ea1fa91..83a61ca6b724 100644
--- a/libs/libgd/Makefile
+++ b/libs/libgd/Makefile
@@ -43,6 +43,8 @@ define Package/libgd-full
   DEPENDS+=+libtiff +libfreetype
   TITLE+=(full)
   VARIANT:=full
+  PROVIDES:=libgd
+  CONFLICTS:=libgd
 endef
 
 define Package/libgd/description/default
